requirements.txt

```diff
@@ -7,6 +7,7 @@ nltk
 numpy
 opacus
 pandas
+portalocker
```

Comment:
When I ran the dynamic layer exchanger example, my install was missing this library.